components/alert/alertPage.css

```diff
@@ -69,4 +69,11 @@ iframe {
 
 .container__component-holder:hover {
     border: 3px solid var(--dui-primary-color-p2-lgt-20pct);
+}
+
+@media only screen and (max-width: 1300px) {
```

Comment:
Suggestion: You can further refactor CSS files with giving your commonly used breakpoint as variable

Reply:
Each page has its own unique breakpoints